src/main/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -157,13 +156,28 @@ default boolean supportsMetricAccumulationLevel() {
         }
     }
 
-    public enum Program implements ProgramInterface {
+    public enum Program implements ProgramInterface, CommandLineParser.ClpEnum {
+
         CollectAlignmentSummaryMetrics {
+
             @Override
             public boolean supportsMetricAccumulationLevel() {
                 return true;
             }
 
+            final String METRICS_EXTENSION = ".alignment_summary_metrics";
+            final String PDF_EXTENSION = ".read_length_histogram.pdf";
+
+            final List<String> OUTPUT_EXTENSIONS = CollectionUtil.makeList(
+                    METRICS_EXTENSION,
+                    PDF_EXTENSION);
+
+            @Override
+            public String getHelpDoc() {
```

Comment:
It looks like this is the same for most (all?) subclasses. If you make `USAGE_SUMMARY` and `OUTPUT_EXTENSIONS` either constructor arguments or abstract methods then the duplicate code can be removed.